build_tools/ci/cpu_comparison/matmul_template/matmul_generator.py

```diff
@@ -14,6 +22,8 @@ def generate_matmul_test(output_fn, input_fn, m, n, k, lhs_rhs_type, acc_type, b
     replace["K"] = k
     replace["TYPE1"] = lhs_rhs_type
     replace["TYPE2"] = acc_type
+    # Only used for Matmul+Trunc via scaling.
+    replace["TYPE3"] = get_higher_order_element_type(acc_type)
```

Comment:
Instead of TYPE1, TYPE2, TYPE3 etc, I think we should start using more descriptive  type names, like for example: `TYPE_MUL_RESULT`. This will scale better towards an increasing number of different tests. No need to change the existing types necessarily, but it would be good to replace `TYPE3` with something more descriptive.

Reply:
Done.